charts/tembo-operator/templates/crd.yaml

```
@@ -152,8 +152,14 @@ spec:
                       - readiness
                       type: object
                     resources:
+                      default:
+                        limits:
+                          cpu: '2'
+                          memory: 2Gi
+                        requests:
+                          cpu: 500m
+                          memory: 512Mi
```

Kinda big default, also I think default should have no oversubscription for memory.

```suggestion
                      default:
                        limits:
                          cpu: '1'
                          memory: 512Mi
                        requests:
                          cpu: 250m
                          memory: 512Mi
```

---

good catch. let me regenerate the CRD

---

100m and 400m for CPU sound good?

---

Yes it looks good